tests/common/nebula_service.py

```diff
@@ -22,7 +22,7 @@ class NebulaService(object):
     def __init__(self, build_dir, src_dir):
         self.build_dir = build_dir
         self.src_dir = src_dir
-        self.work_dir = os.path.join(os.getcwd(), 'build', 'server')
+        self.work_dir = os.path.join(os.getcwd(), 'build', 'server_' + time.strftime("%Y-%m-%dT%H:%M:%S", time.localtime()) )
```

Comment:
Why not use `self.work_dir = os.path.join(build_dir, 'server_.....')`?  If so, `os.mkdir` will work well.

Reply:
I understand that was running the test, where generating the dir. Your understanding is to be generated in a fixed place. it ok for them.